doc-content/enterprise-only/cases/case-management-creating-delivered-milestone-proc.adoc

```diff
@@ -0,0 +1,27 @@
+[id='case-management-create-delivered-milestone-proc']
+= Creating the Delivered to customer milestone
+The condition for this milestone is that a case file variable named `delivered` is `true`. `Adhoc autostart` is not enabled for this milestone. Instead, it is triggered by a signal event after the order has successfully shipped to the customer.
+
+.Procedure
+. In the process designer, expand *Milestone* in the *Object Library* and drag a new milestone on the canvas below the *Send to tracking system* script task.
+. Click the new milestone and in the upper-right corner, click image:cases/left-arrows.png[] to open the *Properties* panel.
+. Input `Milestone 3: Delivered to customer` in the *Name* field.
+. Select `false` from the *Adhoc autostart* menu.
+. Click the down arrow in the *Assignments* field to add a new condition in the *Milestone 3: Delivered to customer* window.
```

Comment:
Minor Question: Are some of the instances of "Click the down arrow" referring to drop-down lists? (I don't know.) If so, perhaps these can be changed to more closely follow the IBM suggestion on p.209?
For example, "From the *Data Type* list, select *String*.".